src/Module.php

```diff
@@ -35,7 +35,7 @@ public function init($event)
         $serviceListener->addServiceManager(
             'FilterManager',
             'filters',
-            'Zend\ModuleManager\Feature\FilterProviderInterface',
+            FilterProviderInterface::class,
```

Comment:
There is another for relax dependencies without break the code.

We could store in this repo the interface `Zend\ModuleManager\Feature\FilterProviderInterface` and setup the file in composer.json as a classmap or as another source of Zend\ModuleManager namespace. Probably we should add lower versions of zend-modulemanager as conflict for to avoid double declaration of the same file.